Examples/mp612-Patient-JORISHANSMAN.xml

```diff
@@ -0,0 +1,33 @@
+<Patient xmlns="http://hl7.org/fhir">
```

Comment:
Ik twijfel toch nog aan de bestandsnamen en -indeling. Een Patient-resource in het zib-package zou normaal gesproken een profiel op nl-core-patient zijn. Omdat je hier iets anders krijgt, is het te verdedigen om de bestandsnaam anders in te steken. Maar wellicht zou het logischer zijn om alle examples voor niet-zib-profielen in een submapje te stoppen met een README erbij.

Reply:
Snap ik, ik zal de voorbeelden in een MP subfolder zetten met een README.
Deze versie van de repo bevat overigens ook nog wel deze niet zib-profielen, wat ook de reden is dat we ze hier willen toevoegen.